src/AlohaKit/AlohaKit.csproj

```diff
@@ -33,4 +33,9 @@
 	  </None>
 	</ItemGroup>
 
+	<ItemGroup>
+	  <PackageReference Update="Microsoft.SourceLink.GitHub" Version="8.0.0" />
+	  <PackageReference Update="Nerdbank.GitVersioning" Version="3.6.133" />
+	</ItemGroup>
```

Comment:
These packages should have the version bumped in the Directory.Build.props where they are defined. Also with .NET 8 we should have the package reference explicitly defined for Maui.Controls.